ui-admin/src/portal/siteContent/designer/editors/ItemSectionsEditor.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useId } from 'react'
+import { ButtonConfig, HtmlSection, ItemSection, SectionConfig } from '@juniper/ui-core'
+import { Button } from 'components/forms/Button'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faChevronDown, faChevronUp, faPlus } from '@fortawesome/free-solid-svg-icons'
+import { ListElementController } from '../components/ListElementController'
+import { ButtonEditor } from './ButtonsEditor'
+import classNames from 'classnames'
+
+/**
+ * Editor for footer item sections
+ */
+export const ItemSectionsEditor = ({ section, updateSection }: {
+    section: HtmlSection, updateSection: (section: HtmlSection) => void
+}) => {
+  const config = JSON.parse(section.sectionConfig || '{}') as SectionConfig
+  const itemSections = config.itemSections as ItemSection[] || []
+  const sectionsContentId = useId()
+  const sectionsTargetSelector = `#${sectionsContentId}`
+
+  return (
+    <div>
+      <div className="pb-1">
+        <button
```

Comment:
this button code/style pattern gets repeated enough below that it seems worth pulling into either a component or helper function, e.g.
`{ renderSectionButton(sectionsTargetSelector, extraClassNames) }`

Reply:
also, do we need a type="button" on these so they don't inadvertently turn into submit buttons in the event that they get surrounded by a form?